tket/src/Transformations/Decomposition.cpp

```diff
@@ -857,7 +865,13 @@ Transform decompose_TK2(const TwoQbFidelities &fid, bool allow_swaps) {
     }
   }
   if (fid.ZZMax_fidelity && fid.ZZPhase_fidelity) {
-    if (*fid.ZZMax_fidelity < (*fid.ZZPhase_fidelity)(.5)) {
+    double ZZPhase_half = std::visit(
```

Comment:
Very unsure about this change, I'm not really sure what this code actually means in the case of a constant value.